hoomd/md/nlist.py

```diff
@@ -181,10 +303,11 @@ def __init__(self,
                  rebuild_check_delay=1,
                  check_dist=True,
                  deterministic=False,
-                 mesh=None):
+                 mesh=None,
+                 default_r_cut=None):
```

Comment:
```suggestion
                 default_r_cut=0.0):
```
Be explicit.

Also, document `default_r_cut` above.